YaqeenDAL/Model/Model.cs

```diff
@@ -210,4 +210,29 @@ public class ResourceLocalization
         [Column(TypeName = "jsonb")]
         public Dictionary<string, string> Translation { get; set; }
     }
-}
\ No newline at end of file
+        public class Country
+    {
+        [Key]
+        public int Id {get; set;}    
+        public string Name { get; set; } 
+        public string AlphaCode { get; set; }
+    }
+    //country state
+           public class CountryState
+    {
+        [Key]
+        public int Id {get; set;}    
+        public string Name { get; set; }
+        public int CountryId { get; set; }
+        public string CountryCode { get; set; }
+        public string CountryName { get; set; }
+        public string StateCode { get; set; }
+        public string? Type { get; set; }
```

Comment:
What's the type?

Reply:
type of country like  (province, territory, state, ...)